compose.go

```diff
@@ -175,3 +179,41 @@ func NewLocalDockerCompose(filePaths []string, identifier string, opts ...LocalD
 
 	return dc
 }
+
+func NewDockerComposeWithReaders(readers []io.Reader) (*dockerCompose, error) {
```

Comment:
Yeah, it's great idea.
I created `WithStackReaders` helper function. And it can be used with `NewDockerComposeWith` contructor.
And of course I remove `NewDockerComposeWithReaders`.
Unit test is updated.

But in `ComposeStackOption.applyToComposeStack` implementation for ComposeStackReaders I did not find another way process errors, so use panic(err). I'm not sure it's good idea, but don't know how to do better.